.gitignore

```diff
@@ -11,7 +12,9 @@ stats.json
 tsDist/
 dist/
 common-dist/
+commons-dist/
```

Comment:
slightly confusing to have both commons-dist and common-dist which are completely different things. Maybe we should rename common-dist to something else? Like vendor-bundles-dist?